functional/platform/nspawn.go

```diff
@@ -302,14 +302,18 @@ func (nc *nspawnCluster) createMember(id string) (m Member, err error) {
 
 		// minimum requirements for running systemd/coreos in a container
 		fmt.Sprintf("mkdir -p %s/usr", fsdir),
+		//fmt.Sprintf("mkdir -p %s/lib", fsdir), //ubuntu
+		//fmt.Sprintf("mkdir -p %s/lib64", fsdir), //ubuntu
 		fmt.Sprintf("cp /etc/os-release %s/etc", fsdir),
+		fmt.Sprintf("echo 'core:x:500:500:CoreOS Admin:/home/core:/bin/bash' > %s/etc/passwd", fsdir),
+		fmt.Sprintf("echo 'core:x:500:' > %s/etc/group", fsdir),
 		fmt.Sprintf("ln -s /proc/self/mounts %s/etc/mtab", fsdir),
 		fmt.Sprintf("ln -s usr/lib64 %s/lib64", fsdir),
 		fmt.Sprintf("ln -s lib64 %s/lib", fsdir),
 		fmt.Sprintf("ln -s usr/bin %s/bin", fsdir),
 		fmt.Sprintf("ln -s usr/sbin %s/sbin", fsdir),
 		fmt.Sprintf("mkdir -p %s/home/core/.ssh", fsdir),
-		fmt.Sprintf("chown -R core:core %s/home/core", fsdir),
+		fmt.Sprintf("chown -R 500:500 %s/home/core", fsdir),
```

Comment:
I would suggest to separate these requirements on their own, not hard code them inside the code, just a separate function that does the setup for us in its own file ? @kayrus you could open a PR later for this one perhaps ?